src/js/components/FormField/FormField.js

```diff
@@ -134,12 +134,14 @@ const Input = ({ component, disabled, invalid, name, onChange, ...rest }) => {
   );
 };
 
-const debounce = (func, wait) => {
+const debounce = (func, wait, fieldRef) => {
```

Comment:
Instead of making debounce() know about the ref, what about changing it to be a hook that will clear the timer automatically when the component is unmounted?

Reply:
Good idea, I updated to use a hook in the latest commit

Reply:
A suggestion to tighten it up a bit:
```
const useDebounce = () => {
  const [func, setFunc] = useState();

  useEffect(() => {
    const timer = setTimeout(() => func(), theme.global.debounceDelay);
    return () => clearTimeout(timer);
  }, [func]);
  
  return setFunc;
};

...
const debounce = useDebounce();
...
if (contextOnChange) debounce(() => contextOnChange(event)));
```
untested, but perhaps you get the idea?